### PR TITLE
Fixed #33455 -- Improved error message when selenium is not installed.

### PR DESCRIPTION
**Pull request by My-Name-Is-Nabil**
Had a couple of ideas in mind on how to check if selenium is installed, went with the idea suggested in the ticket comments (importing selenium).

### CODE REVIEW

File: tests/runtests.py [hunk 1]
```diff
@@ -321,6 +321,12 @@ class ActionSelenium(argparse.Action):
     Validate the comma-separated list of requested browsers.
     """
     def __call__(self, parser, namespace, values, option_string=None):
+        # Check if selenium is not installed.
+        try:
+            import selenium  # noqa
+        except ImportError:
+            print("Selenium needs to be installed to run selenium tests.")
+            sys.exit(1)
```
Comment (smithdc1) [inline tests/runtests.py:329]: Maybe 
```suggestion
    # check if selenium is installed.
    import selenium  # noqa
```
This then crashes with following error message which I think is good enough to work out what's going on. I'm not sure we need a custom error message. 🤔 
```ModuleNotFoundError: No module named 'selenium'```

**Comment by My-Name-Is-Nabil** [comment]
Yeah that makes sense, I changed it.

**Comment by jacobtylerwalls** [comment]
We could also try:

```diff
diff --git a/tests/runtests.py b/tests/runtests.py
index 2ec5d32969..1b98617053 100755
--- a/tests/runtests.py
+++ b/tests/runtests.py
@@ -325,7 +325,9 @@ class ActionSelenium(argparse.Action):
         for browser in browsers:
             try:
                 SeleniumTestCaseBase.import_webdriver(browser)
-            except ImportError:
+            except ImportError as e:
+                if 'selenium.webdriver' not in e.msg:
+                    raise e
                 raise argparse.ArgumentError(self, "Selenium browser specification '%s' is not valid." % browser)
```

**Comment by My-Name-Is-Nabil** [comment]
I had this idea, but in other places the import method is used (such as /tests/validators/tests.py). Also this syntax isn't really readable. I am, however, open to both ideas.

File: tests/runtests.py [hunk 2]
```diff
@@ -321,6 +321,8 @@ class ActionSelenium(argparse.Action):
     Validate the comma-separated list of requested browsers.
     """
     def __call__(self, parser, namespace, values, option_string=None):
+        # Check if selenium is installed.
+        import selenium  # noqa
```
Comment (felixxm) [inline tests/runtests.py:325]: I'd raise ``ImproperlyConfigured`` as in other cases:
```suggestion
        try:
            import selenium
        except ImportError as e:
            raise ImproperlyConfigured(f'Error loading selenium module: {e}')
```

**Review by felixxm — COMMENTED** [review]
@My-Name-Is-Nabil Thanks for this patch :+1:

**Comment by felixxm** [comment]
@My-Name-Is-Nabil Thanks :+1:

**Pull request merged**